src/proto/feature_flag.proto

```diff
@@ -41,4 +41,17 @@ package otbr;
 message FeatureFlagList {
   // Whether to enable the NAT64 feature.
   optional bool enable_nat64 = 1 [default = false];
+  // Whether to enable detailed logging.
+  optional bool enable_detailed_logging = 2 [default = false];
+  // Set specific detailed logging level, default = OTBR_LOG_INFO
+  // Mapping from detailed_logging_level number to otbrLogLevel:
+  //   0 - OTBR_LOG_EMERG
+  //   1 - OTBR_LOG_ALERT
+  //   2 - OTBR_LOG_CRIT
+  //   3 - OTBR_LOG_ERR
+  //   4 - OTBR_LOG_WARNING
+  //   5 - OTBR_LOG_NOTICE
+  //   6 - OTBR_LOG_INFO
+  //   7 - OTBR_LOG_DEBUG
```

Comment:
Why do we want to introduce these new log levels?

Reply:
We would like to be able to specify detailed logging level from feature flags. There doesn't seem to be a way to directly use the otbrLogLevel definition in proto files or the generated cc/h file, so I added this translation.

Reply:
I wrongly thought these were different from OTBR's enums. 

On the other hand, why do we still need to manually map the integer to enum `otbrLogLevel`? Why not just assign the integer to the enum?

Reply:
"assign the integer to the enum" do you mean something like casting the int to the enum? I prefer the mapping as it's more transparent, and it separates the definition in otbr and proto. The proto int number may be updated from feature flag server side. Say if the otbrLogLevel numbers changed, e.g. we somehow decided to add a otbr log level in the middle, then casting the int will require us to modify the proto file and also the feature flags server side.

Reply:
Proto enum should be a more clear way to address such enum value changes.